validator/jobroleValidator.ts

```diff
@@ -5,5 +5,13 @@ module.exports.validateJobrole = function (jobrole: JobRole): string {
         return "Job role greater than 50 characters";
     }
 
+    if (jobrole.jobSpecification.length > 70) {
```

Comment:
I don't think we need a validator as this data is pre-defined. It will be important to include a validator when you get to the add ticket. However, for now we don't need to checks this.

Reply:
This has been fixed